test/jdk/java/net/httpclient/HttpRequestBodyPublishers/OfFileTest.java

```diff
@@ -49,7 +49,7 @@
 
 /*
  * @test
- * @bug 8364733
+ * @bug 8226303 8235459 8358688 8364733
```

Comment:
Copied 8226303 and 8358688 from `RelayingPublishers`, and 8235459 from `FilePublisherTest` to preserve the association.